test/e2e/pages/Project.page.js

```diff
@@ -0,0 +1,130 @@
+class ProjectPage {
+  open(){
+    browser.url('https://asana.com/es#login');
+  }
+
+  get inputEmail(){
+    return $('#login-email-modal-login');
+  }
+
+  get inputPassword(){
+    return $('#login-password-modal-login');
+  }
+
+  get buttonLogin(){
+    return $('#login-submit-modal-login');
+  }
+
+  get linkNewProject(){
+    return $('//a[contains(@class,\'SidebarTeamDetailsProjectsList-emptyTeamCreateProjectLink\')]');
+  }
+
+  get templateNewProject(){
+    return $('div[title=\'Proyecto en blanco\']');
+  }
+
+  get typeNewProject(){
+    return $('(//div[contains(@class,\'ProjectLayoutSelector-layoutOptionLabel\')])[2]');
+  }
+
+  get inputProjectName(){
+    return $('#new_project_dialog_content_name_input');
+  }
+
+  get buttonSaveProject(){
+    return $('//div[contains(@class,\'submitButton\')]');
+  }
+
+  get nameProjectCreated(){
+    return $('//h1[contains(@class,\'Typography--truncate TopbarPageHeaderStructure-title\')]');
+  }
+
+  get firstCard(){
+    return $('(//div[contains(@class,\'BoardCard\')])[1]');
+  }
+
+  get linkNewColumn(){
+    return $('//a[contains(@class,\'BoardNewColumn-addColumnLink\')]');
+  }
+
+  get addNewTask(){
+    return $('(//a[contains(@class,\'CardButton\')])[4]');
+  }
+
+  get titleNewColumnForAssertion(){
+    return $('(//span[contains(@class,\'BoardColumnHeaderTitle\')])[4]');
+  }
+
+  get titleNewCardForAssertion(){
+    return $('(//div[contains(@class,\'BoardCard-name\')])[4]');
+  }
+
+  get textAreaNewCardTitle(){
+    return $('//textarea[contains(@class,\'simpleTextarea\')]');
+  }
+
+  get textForNewColumn(){
+    return $('//input[contains(@class,\'BoardNewColumnForm-input\')]');
+  }
+
+  get buttonMenuProject(){
+    return $('//a[contains(@class,\'PageHeaderDropdownMenuButton\')]');
+  }
+
+  get buttonMenuDeleteProject(){
+    return $('(//span[contains(@class,\'MenuItem-label\')])[10]');
```

Comment:
same observation above, there are too many elements with the same selector, so there must be another way to get it